concrete-core-fixture/src/fixture/lwe_ciphertext_cleartext_discarding_multiplication.rs

```diff
@@ -0,0 +1,179 @@
+use crate::fixture::Fixture;
+use crate::generation::prototyping::{
+    PrototypesCleartext, PrototypesLweCiphertext, PrototypesLweSecretKey, PrototypesPlaintext,
+};
+use crate::generation::synthesizing::{SynthesizesCleartext, SynthesizesLweCiphertext};
+use crate::generation::{IntegerPrecision, Maker};
+use crate::raw::generation::RawUnsignedIntegers;
+use crate::raw::statistical_test::assert_noise_distribution;
+use concrete_commons::dispersion::{DispersionParameter, LogStandardDev, Variance};
+use concrete_commons::numeric::UnsignedInteger;
+use concrete_commons::parameters::LweDimension;
+use concrete_core::prelude::{
+    CleartextEntity, LweCiphertextCleartextDiscardingMultiplicationEngine, LweCiphertextEntity,
+};
+
+/// A fixture for the types implementing the `LweCiphertextCleartextDiscardingMultiplicationEngine`
+/// trait.
+pub struct LweCiphertextCleartextDiscardingMultiplicationFixture;
+
+#[derive(Debug)]
+pub struct LweCiphertextCleartextDiscardingMultiplicationParameters {
+    pub noise: Variance,
+    pub lwe_dimension: LweDimension,
+}
+
+#[allow(clippy::type_complexity)]
+impl<Precision, Engine, InputCiphertext, Cleartext, OutputCiphertext>
+    Fixture<Precision, Engine, (InputCiphertext, Cleartext, OutputCiphertext)>
+    for LweCiphertextCleartextDiscardingMultiplicationFixture
+where
+    Precision: IntegerPrecision,
+    Engine: LweCiphertextCleartextDiscardingMultiplicationEngine<
+        InputCiphertext,
+        Cleartext,
+        OutputCiphertext,
+    >,
+    InputCiphertext: LweCiphertextEntity,
+    Cleartext: CleartextEntity,
+    OutputCiphertext: LweCiphertextEntity<KeyDistribution = InputCiphertext::KeyDistribution>,
+    Maker: SynthesizesCleartext<Precision, Cleartext>
+        + SynthesizesLweCiphertext<Precision, InputCiphertext>
+        + SynthesizesLweCiphertext<Precision, OutputCiphertext>,
+{
+    type Parameters = LweCiphertextCleartextDiscardingMultiplicationParameters;
+    type RepetitionPrototypes = (
+        <Maker as PrototypesLweSecretKey<Precision, InputCiphertext::KeyDistribution>>::LweSecretKeyProto,
+        <Maker as PrototypesCleartext<Precision>>::CleartextProto,
+    );
+    type SamplePrototypes = (
+        <Maker as PrototypesPlaintext<Precision>>::PlaintextProto,
+        <Maker as PrototypesLweCiphertext<Precision, InputCiphertext::KeyDistribution>>::LweCiphertextProto,
+        <Maker as PrototypesLweCiphertext<Precision, InputCiphertext::KeyDistribution>>::LweCiphertextProto,
+    );
+    type PreExecutionContext = (InputCiphertext, Cleartext, OutputCiphertext);
+    type PostExecutionContext = (InputCiphertext, Cleartext, OutputCiphertext);
+    type Criteria = (Variance,);
+    type Outcome = (Precision::Raw, Precision::Raw);
+
+    fn generate_parameters_iterator() -> Box<dyn Iterator<Item = Self::Parameters>> {
+        Box::new(
+            vec![LweCiphertextCleartextDiscardingMultiplicationParameters {
```

Comment:
In the second test in private, there was randomness on the parameters' values (LWE dimension and number of samples). Maybe here we could at least be testing for other LWE dimensions?